aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -77,6 +77,30 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> Default
 }
 
 impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGroupValue<T, V> {
+    fn set_base_values(
+        &mut self,
+        shifted_idx: ShiftedTxnIndex,
+        incarnation: Incarnation,
```

Comment:
The incarnation is given as input to `set_base_values` function here. But the incarnation is not given as input to `provide_base_value` function in versioned_data.rs. Any reason for that?